src/CacheCow.Client.RedisCacheStore/CacheCow.Client.RedisCacheStore.csproj

```diff
@@ -1,15 +1,17 @@
 ﻿<Project Sdk="Microsoft.NET.Sdk">
 	<PropertyGroup>
-		<TargetFrameworks>netstandard2.0;net452</TargetFrameworks>
+		<TargetFrameworks>netstandard2.0;net461</TargetFrameworks>
 		<TargetFrameworks Condition="'$(OS)' != 'Windows_NT'">netstandard2.0</TargetFrameworks>		
 		<Summary>Redis Storage for HTTP Client Caching</Summary>
 		<AssemblyName>CacheCow.Client.RedisCacheStore</AssemblyName>
 		<PackageId>CacheCow.Client.RedisCacheStore</PackageId>
+		<GeneratePackageOnBuild>true</GeneratePackageOnBuild>
```

Comment:
Also this is not needed.

Reply:
Oh sorry - that was me building the package locally so that I could test to make sure it did what we needed. Do you want me to make another pull request to remove that?